profiles/common/modules/custom/tmgmt_poetry/inc/tmgmt_poetry.webservice.inc

```diff
@@ -0,0 +1,470 @@
+<?php
+/**
+ * @file
+ * Webservice definition and behaviour.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function tmgmt_poetry_menu() {
+  $items = array();
+  $items['tmgmt_poetry/service_callback'] = array(
+    'page callback' => '_tmgmt_poetry_service_callback',
+    'access callback' => TRUE,
+  );
+  $items[drupal_get_path("module", "tmgmt_poetry") . "/wsdl/PoetryIntegration.wsdl"] = array(
+    'page callback' => '_tmgmt_poetry_wsdl',
+    'access callback' => TRUE,
+  );
+  return $items;
+}
```

Comment:
Just a note: main functions and hook implementations should always be put in the module file. They should not be put in include files. Include files should contain secondary code that can be included at runtime.

The only way that this will work is if the include file is hard included in the module file using `require_once()` which defeats the purpose of using include files using `module_load_include()`.